doc/api/magic-plot.md

```diff
@@ -103,7 +103,7 @@ WHERE island = 'Biscoe'
 ```
 
 ```{code-cell} ipython3
-%sqlplot boxplot --table biscoe --column body_mass_g --with biscoe
+%sqlplot boxplot --table biscoe --column body_mass_g
```

Comment:
add some context here, mention that since we ran `--save biscoe`, we can pass it as an argument to `--table` and it'll work since jupysql will auto-generate the CTE

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/871/files#diff-d2bac2aa48fbf4cb4aee34358d99fb9653fe443d5d3359ea2848dd36bdaa2f3bR105)